src/bookmarks.rs

```diff
@@ -0,0 +1,95 @@
+use crate::config::Flattenable;
```

Comment:
This should probably be placed under `src/config/default` or `src/config/bookmark`